modules/web/src/de/diedavids/cuba/dataimport/web/importwizard/ImportWizard.groovy

```diff
@@ -99,23 +103,60 @@ class ImportWizard extends AbstractWindow {
     @Inject
     ImportAttributeMapperCreator importAttributeMapperCreator
 
+    ImportConfiguration defaultImportConfiguration;
+
+    @Inject
+    Button toStep3;
+
     @Override
     void init(Map<String, Object> params) {
 
         entityLookup.setOptionsMap(entityClassSelector.entitiesLookupFieldOptions)
 
         importConfigurationDs.setItem(createImportConfiguration())
-
         initEntityClassPropertyChangeListener()
         initReusePropertyChangeListener()
         initImportFileHandler()
         initImportFileParser()
+        initTitles();
+        configLookup.addListener(new ValueListener() {
+
+            void valueChanged(Object source, String property, Object prevValue, Object value) {
+                if (value) {
+                    def vv = (ImportConfiguration) value;
+                    /*def importConfiguration = metadata.create(ImportConfiguration)
+                    importConfiguration.dateFormat = vv.dateFormat
+                    importConfiguration.booleanTrueValue = vv.booleanTrueValue
+                    importConfiguration.booleanFalseValue = vv.booleanFalseValue
+                    importConfiguration.importAttributeMappers = vv.importAttributeMappers
+                    */
+
+                    importConfigurationDs.setItem(vv)
+                } else {
+                    importConfigurationDs.setItem(defaultImportConfiguration)
+                    entityLookup.setValue(importConfigurationDs.getItem().entityClass)
+                }
+
+            }
+        });
+
+
+    }
+
+    void initTitles() {
+        try {
+            def tabs = wizardAccordion.getTabs()
+            tabs.each {
+                it.caption = formatMessage(it.name.concat("Title"), "")
+            }
+        } catch (Exception e) {
```

Comment:
we should not do that - the exception gets lost in space. What can happen an Exception here?